Manual.md

```diff
@@ -1533,7 +1533,11 @@ The `python3-pep517` build style provides means to build python packages that pr
 definition compliant with [PEP 517](https://www.python.org/dev/peps/pep-0517/) without a traditional
 `setup.py` script.
 
-Python packages that rely on `python3-setuptools` should generally map `setup_requires`
+Python packages that use [Poetry](https://github.com/python-poetry/poetry-core)
+as a build backend should specify the `python3-pep517` build style, add
+`python3-poetry-core` to `hostmakedepends` and approximately map
+`tool.poetry.dependencies` entries in `pyproject.toml` to `depends` in the
+template.  Python packages that rely on `python3-setuptools` should generally map `setup_requires`
```

Comment:
I think the poetry stuff, being new and less used (at least for now), should go below the `setuptools` instructions.